src/test/java/PlaySongTests.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class PlaySongTests extends BaseTest {
+    @Test
+    public void playSong()
+    //throws InterruptedException
+    {
+        // GIVEN user is on the login page ("https://bbb.testpro.io/");
+        driver.get(url);
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226535959-480c9b02-7833-4944-9347-9595948b7e8d.png)